definitions/scenarios/content.rb

```diff
@@ -43,11 +43,8 @@ class Switchover < ContentBase
       end
 
       def compose
-        # FIXME: remove this condition for the 6.10 upgrade scenario
-        if Procedures::Content::Switchover.present?
-          add_step(Procedures::Content::Switchover)
+          add_step_with_context(Procedures::Content::Switchover)
```

Comment:
Removed the present? condition as procedure is present now, and added step using context so confine block will be used before running the procedure.